scripts/start.sh

```diff
@@ -40,10 +40,11 @@ function setup_sqlflow_server() {
   DS="mysql://root:root@tcp(${SQLFLOW_MYSQL_HOST}:${SQLFLOW_MYSQL_PORT})/?maxAllowedPacket=0"
   echo "Connect to the datasource ${DS}"
   # Start sqlflowserver
-  sqlflowserver --datasource=${DS}
+  source activate sqlflow-dev && sqlflowserver --datasource=${DS}
 }
 
 function setup_sqlflow_notebook() {
+  cd /workspace
```

Comment:
shall we `cd ..` at the end of this function? Just in case, later on, we wanna add command after `setup_sqlflow_notebook`.